gremlin-go/driver/client.go

```diff
@@ -29,38 +30,43 @@ type ClientSettings struct {
 	LogVerbosity    LogVerbosity
 	Logger          Logger
 	Language        language.Tag
+	AuthInfo        *AuthInfo
+	TlsConfig       *tls.Config
```

Comment:
Should these be pointers in the settings?

This would effectively allow someone to mutate these after instantiating the Client.